com.unity.render-pipelines.high-definition/Editor/Material/Decal/ShaderGraph/DecalSubTarget.cs

```diff
@@ -448,11 +457,11 @@ static class DecalRenderStates
                 { RenderState.Blend("Blend 0 SrcAlpha OneMinusSrcAlpha, Zero OneMinusSrcAlpha Blend 1 SrcAlpha OneMinusSrcAlpha, Zero OneMinusSrcAlpha Blend 2 SrcAlpha OneMinusSrcAlpha, Zero OneMinusSrcAlpha") },
                 { RenderState.ZTest(ZTest.LEqual) },
                 { RenderState.ZWrite(ZWrite.Off) },
-                { RenderState.ColorMask(s_DecalColorMasks[4]) },
+                { RenderState.ColorMask("ColorMask BA 2") },
```

Comment:
is it expected?

Reply:
harcoded mask value here

Reply:
There is the same hardcoded value in Decal.shader If I'm not mistaken